Unable to load policy upgraded to latest version of `casbin` version.

### Description
- Unable to load policy in casbin version `v2.55.1`.

### Issue
```
# github.com/casbin/casbin-pg-adapter
vendor/github.com/casbin/casbin-pg-adapter/adapter.go:393:29: cannot use persist.LoadPolicyLine (type func(string, model.Model) error) as type func(string, model.Model) in argument to a.loadFilteredPolicy
```

--- a/.gitignore
+++ b/.gitignore
@@ -39,3 +39,4 @@ _testmain.go
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+vendor/

--- a/adapter.go
+++ b/adapter.go
@@ -199,7 +199,10 @@ func (a *Adapter) LoadPolicy(model model.Model) error {
 	}
 
 	for _, line := range lines {
-		persist.LoadPolicyLine(line.String(), model)
+		err := persist.LoadPolicyLine(line.String(), model)
+		if err != nil {
+			return err
+		}
 	}
 
 	a.filtered = false
@@ -423,7 +426,7 @@ func buildQuery(query *orm.Query, values []string) (*orm.Query, error) {
 	return query, nil
 }
 
-func (a *Adapter) loadFilteredPolicy(model model.Model, filter *Filter, handler func(string, model.Model)) error {
+func (a *Adapter) loadFilteredPolicy(model model.Model, filter *Filter, handler func(string, model.Model) error) error {
 	if filter.P != nil {
 		lines := []*CasbinRule{}
 

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/casbin/casbin-pg-adapter
 go 1.14
 
 require (
-	github.com/casbin/casbin/v2 v2.36.1
+	github.com/casbin/casbin/v2 v2.55.1
 	github.com/go-pg/pg/v10 v10.9.1
 	github.com/mmcloughlin/meow v0.0.0-20181112033425-871e50784daf
 	github.com/stretchr/testify v1.7.0

